### PR TITLE
Regenerate on `sdk-1.3.243.0`

### PR DESCRIPTION
**Pull request by MarijnS95**
Something I had sitting on a local branch for quite some time; certain ops might need to be adjusted as with the previous "update SPIRV-Headers" PR, and some issues need to be addressed with overlapping `match` ranges for stabilized/promoted `Op` constants that shadow each other.

### CODE REVIEW

**Comment by BeastLe9enD** [comment]
@MarijnS95 I'm currently maintaining a private branch of rspirv since I'm adding support for mesh shading to rust-gpu which depends on it. I also made a minor change to rspirv to allow EmitMeshTasksEXT being recognized as an aborting op. May I PR my branch into your regen-1.3 branch? Because it makes more sense than having two separate PR's for rspirv :)

**Comment by MarijnS95** [comment]
@BeastLe9enD I'd rather see this PR land separately, so that your changes can be reviewed separately; unless they are necessary to properly map the new bindings within this PR, by adding edge-cases for new ops. Added the `EmitMeshTasksEXT` for that case to make this PR more complete!

**Comment by BeastLe9enD** [comment]
@MarijnS95 Nice, thank you!

File: rspirv/README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ First add to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-rspirv = "0.11.0+sdk-1.2.198"
+rspirv = "0.11.0"
```
Comment (MarijnS95) [inline rspirv/README.md:37]: This should also be gone automatically on the next `cargo-release` now that https://github.com/crate-ci/cargo-release/issues/308#issuecomment-905489443 is in and `{{version}}` excludes metadata, which is the substitute for https://github.com/crate-ci/cargo-release/pull/312 / #213.

File: rspirv/sr/autogen_ops.rs [hunk 2]
```diff
@@ -47,13 +47,20 @@ pub enum Branch {
         size: u32,
     },
     TerminateInvocation,
+    DemoteToHelperInvocation,
```
Comment (MarijnS95) [inline rspirv/sr/autogen_ops.rs:50]: Perhaps this belongs with one of the block terminators in `rspirv/grammar/reflect.rs`.

Comment (msiglreith) [reply rspirv/sr/autogen_ops.rs:50]: Based on the specification wording I would exclude it
>Unlike the [OpTerminateInvocation](https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#OpTerminateInvocation) instruction, this does not necessarily terminate the invocation which might be needed for derivative calculations. It is not considered a flow control instruction (flow control does not become non-uniform) and does not terminate the block.

**Review by msiglreith — APPROVED** [review]

File: rspirv/sr/autogen_ops.rs (hunk 2 above)
Comment (MarijnS95) [reply rspirv/sr/autogen_ops.rs:50]: Oh cool, thanks! I had a hard time finding the certain ops such as `EmitMeshTasksEXT`, `SetMeshOutputsEXT`, `HitObject*NV`, `ReorderThead*NV` andsoforth in [the spec](https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html) but at least this one is properly documented.

**Comment by MarijnS95** [comment]
@msiglreith Any other ops (in particular those not documented in https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html) that need special treatment or does it all look good to you?

**Comment by msiglreith** [comment]
No, looks good to me, thanks!
Quickly [searching](https://github.com/search?q=repo%3AKhronosGroup%2FSPIRV-Registry+Termination+Instructions&type=code) through the registry didn't reveal any new additions

**Pull request merged**